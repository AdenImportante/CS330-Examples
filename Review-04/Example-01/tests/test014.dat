#! /bin/sh

# Repeat test013 with attention to formatting

# Consider all whitespace and letter-case (upper/lower)
./storage test013.in 4 