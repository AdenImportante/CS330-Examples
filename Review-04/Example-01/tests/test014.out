Processing Log:
 (S) Speed-II-Potion
 (S) PotatoCamera

Player Storage Summary:
 -Used  50% of 4 slots
  Nme: Speed-II-Potion
  Eft: Spd*2
  Use: 1
  Qty: 1

  Nme: PotatoCamera
  Eft: Focus-50%
  Use: 99
  Qty: 1

